msbuild/Xamarin.Localization.MSBuild/Xamarin.Localization.MSBuild.csproj

```diff
@@ -6,20 +6,18 @@
   <PropertyGroup>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
   </PropertyGroup>
-  <ItemGroup>
-    <EmbeddedResource Update="MSBStrings.resx">
-      <Generator>PublicResXFileCodeGenerator</Generator>
-      <LastGenOutput>MSBStrings.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
-  </ItemGroup>
   <ItemGroup>
     <Compile Update="MSBStrings.Designer.cs">
       <DependentUpon>MSBStrings.resx</DependentUpon>
     </Compile>
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="MSBStrings.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>

```

Comment:
I have this from a local experiment I was doing: https://gist.github.com/rolfbjarne/7987f95c6751823eb4f7edf25c88cf94 - the hope is to make it so that the  *designer.cs file is re-generated when the resx file is changed in a text editor and we rebuild from the command line. That might be nice to get for both resx files we have (mtouch.csproj and Xamarin.Localization.MSBuild.csproj).